src/main/java/hudson/remoting/Launcher.java

```diff
@@ -568,6 +569,20 @@ private SSLSocketFactory getSSLSocketFactory()
         }
     }
 
+    private boolean parameterReferencesFile(String parameter) {
+        return parameter.charAt(0) == '@';
```

Comment:
is there any chance of getting a false positive here?

Reply:
No, I don't believe so. Secrets are encoded in hexadecimal (AES 128b secret key).